mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairCards.tsx

```diff
@@ -375,6 +185,245 @@ export function PoolPairCards ({
   )
 }
 
+interface PoolCardProps {
+  item: DexItem<WalletToken | PoolPairData>
+  expandedCardIds: string[]
+  setExpandedCardIds: (ids: string[]) => void
+  isFavouritePoolpair: (id: string) => boolean
+  setFavouritePoolpair: (id: string) => void
+  onAdd: (data: PoolPairData) => void
+  onRemove: (data: PoolPairData) => void
+  onSwap: (data: PoolPairData) => void
+  type: 'your' | 'available'
+  index: number
+}
+
+const PoolCard = ({
+  index,
+  item,
+  expandedCardIds,
+  isFavouritePoolpair,
+  setFavouritePoolpair,
+  type,
+  onAdd,
+  onRemove,
+  onSwap,
+  setExpandedCardIds
+}: PoolCardProps): JSX.Element => {
+  const { getTokenPrice } = useTokenPrice()
+  const { calculatePriceRates } = useTokenBestPath()
```

Comment:
Can you explain what this method does?

Reply:
In pool pair cards and Composite Swap screen we are displaying `aToBPrice` and `bToAPrice`, using this method we are calculating the price. 
this method uses `getBestPath` to get best path for conversion and on the basis of the output we calculate aToB and btoA price